docs/docusaurus/src/components/OverviewCard/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './styles.module.scss';
+
+export default function OverviewCard(
+    {
+        title,
+        children
+    }
+) {
+    return (
+        <div className={styles.overviewCard} id="overviewCard">
+            <div className={styles.overviewCard__inner_container}>
+                <h1 className={styles.overviewCard__title}>{title}</h1>
+                <h4 className={styles.overviewCard__description}>{children}</h4>
+            </div>
+        </div>
```

Comment:
Can we use semantic tags for the card? also I don't think you need the nested div the `overviewCard__inner_container` class.

Reply:
I took out the inner container and canged the container div for a `header` tag, as it's the header of the doc page, do you think that's good to go or is there any other change you would expect me to do here?